### DISCUSSION TITLE
Download only audio tracks?

### DISCUSSION THREAD

**Discussion by claustro** [Tyrrrz/YoutubeDownloader#480] [Q&A]
If I remember correctly in  previous version was possible to download only the audio track. Is it possible?
Героям слава!

**Comment by Tyrrrz** [comment]
Yes, you can scroll down to choose an audio-only format to download in

**Comment by claustro** [comment]
Thank you for your answer maybe low level of glucose in my brain, I missed the, even evident, option. Thank you!
